src/Paths/Directory.ts

```diff
@@ -24,8 +24,8 @@ export const root = "" as Directory;
  * |: Invalid because it might have a use in the terminal in the future.
  * #: Invalid because it might have a use in the terminal in the future.
  * (quote marks): Invalid to avoid conflict with quote marks used in the terminal.
- * (space): Invalid to avoid confusion with terminal command separator */
-const invalidCharacters = ["/", "*", "?", "[", "]", "!", "\\", "~", "|", "#", '"', "'", " "];
+ * (whitespace): Invalid to avoid confusion with terminal command separator */
+const invalidCharacters = ["/", "*", "?", "[", "]", "!", "\\", "~", "|", "#", '"', "'", "\\s"];
```

Comment:
I don't think this will work as-is, because of how `oneValidCharacter` is constructed via escapeRegExp in the next line.

Reply:
Oh yeah, it's failing tests.